### DISCUSSION TITLE
Choice for  pavement and road divider

### DISCUSSION THREAD

**Discussion by hoizan** [streetcomplete/StreetComplete#4498] [Feedback]
Always face ambiguity when asked about whether there is  pavement on both sides of a road/carriageway. In SG, most of the dual directional roads have pavement on one side and a divider between the 2 carriageways. The divider can be road line drawn, or a raised divider(island).  Please improve the questionnaire for answer pavement or divider.

**Comment by matkoniecz** [comment]
How it can be improved? Have you tried sidewalk overlay?

In case of a dual carriageway with pavement on both sides of a road (no pavement between carriageways):

it is mapped as two separate carriageways, each having sidewalk on outer side and not having them on inner side (sidewalk can be mapped as carriageway property or as separate geometry)


In case of a dual carriageway with pavement on both sides of a road (pavement also between carriageways - happens rarely):

it is mapped as two separate carriageways, each having sidewalk on both sides.

**Reply by hoizan** [reply]
Thank you for the clarification. Indeed, I intuitively using that rules to complete the quests in Street Complete. Sorry, I'm only using Street Complete to help up with OSM. I  not aware of the sidewalk overlay thing. This case can be closed if there is no classification for road divider and deems not necessary in OSM.
